### DISCUSSION TITLE
Have anyone configured the telegram and worked?

### DISCUSSION THREAD

**Discussion by mshaabaan** [chaskiq/chaskiq#529] [Q&A]
Can anyone help me with telegram integrations
i added the token from botfather in chaskiq and i dont receive anything

**Comment by michelson** [comment]
are you working in a production app? or a local app? if it's local, have you configured the app so it's possible to access their address from outside?

**Reply by mshaabaan** [reply]
it is production on Azure, the FQDN is reachable from the internet

**Reply by mshaabaan** [reply]
Now it is working..

I configured SSL reverse proxy nginx
switched from sidekiq prod to dev
and started working

**Reply by michelson** [reply]
that's great, but why dev env? if you are using it for production, the dev en will be a slow setup

**Reply by mshaabaan** [reply]
I dont know why,

If i switched to Sidekiq Prod, it won't work

i will switch to confirm

**Reply by mshaabaan** [reply]
It is working :( Lol

I dont know what happened.
I'm working with Azure, I had a lot of challenges to configure Redis in your app to use secret key
and also i added nginx in front of your app with Let's encrypt SSL
working with FB and Telegram

just a question
Chaskiq only gets display name and userid in facebook, nothing else, is that normal?

also is there any way to integrate more than one facebook page.
last thing, Can I have several inboxes based on the channel?
